src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,31 +1,41 @@
+import { Card, WithStyles } from '@material-ui/core';
+import CardMedia from '@material-ui/core/CardMedia';
+import { withStyles } from '@material-ui/core/styles';
 import * as React from 'react';
 import { RouteComponentProps } from 'react-router-dom';
 import { Movie } from '../App/App';
-import './MovieCard.scss';
+import movieCardStyles from './MovieCard.styles';
 
 interface Props extends RouteComponentProps<any> {
     handleMovieCardClick: (title: string) => void;
     movie: Movie;
 }
 
-export default React.memo(({ handleMovieCardClick, movie, history }: Props) => {
+type PropsWithStyles = Props & WithStyles<'card' | 'media' | 'titleDate' | 'title' | 'date' | 'genres'>;
+
+function MovieCard({ handleMovieCardClick, movie, history, classes }: PropsWithStyles) {
     const _handleMovieCardClick = () => {
         history.push(`/film/${movie.id}`);
         handleMovieCardClick(movie.title);
     };
 
     return (
-        <ul
-            className="movie-card"
+        <Card
+            className={classes.card}
             onClick={_handleMovieCardClick}
-            role="presentation"
         >
-            <li><img src={movie.poster_path} width="250px" alt="{props.title}" draggable={false} /></li>
-            <li className="title-date">
-                <span className="title">{movie.title}</span>
-                <span className="date">{movie.release_date.split('-')[0]}</span>
-            </li>
-            <li className="genres">{movie.genres.join(' & ')}</li>
-        </ul>
+            <CardMedia
+                className={classes.media}
+                image={movie.poster_path}
+                title={movie.title}
+            />
+            <div className={classes.titleDate}>
+                <span className={classes.title}>{movie.title}</span>
+                <span className={classes.date}>{movie.release_date.split('-')[0]}</span>
+            </div>
+            <div className={classes.genres}>{movie.genres.join(' & ')}</div>
+        </Card>
     );
-});
+}
+
+export default React.memo(withStyles(movieCardStyles)(MovieCard));
```

Comment:
Prefer composition inside `index.ts`, for test you can still use `identity-obj-proxy` for `classes` property.

Reply:
done